packages/cardano-services/src/NetworkInfo/DbSyncNetworkInfoProvider/mappers.ts

```diff
@@ -0,0 +1,62 @@
+import { Cardano, NetworkInfo, ProviderError, ProviderFailure, TimeSettings } from '@cardano-sdk/core';
+import { GenesisData } from './types';
+import JSONbig from 'json-bigint';
+import fs from 'fs';
+import path from 'path';
+interface ToNetworkInfoInput {
```

Comment:
I'm surprised our linting rules are not mandating a blank line after the imports here :thinking:

Reply:
Please make a ticket to do it separately. I don't really see a problem with letting this through now, given the lint rules say it's OK.